streams/create-stream.js

```diff
@@ -59,214 +58,32 @@ async function main() {
     }
     // STEP 4 - find level. Level with same name should exist in default model.
     const levelKey = Encoding.toFullKey(targetRoom[QC.Level], true);
-    const levelDetails = await getElementData(token, targetRoomModelId, levelKey);
-    const levels = await getLevels(token, defaultModel.modelId);
+    const levelDetails = await client.getElement(targetRoomModelId, levelKey);
+    const levels = await client.getLevels(defaultModel.modelId);
     const targetLevel = levels.find(l => l[QC.Name] === levelDetails[QC.Name]);
 
     if (!targetLevel) {
         throw new Error(`Level ${levelDetails[QC.Name]} doesn't exist`);
     }
     // STEP 5 - create new stream. First step is to encode keys for references. In our case host element and room are same.
-    const parentXref = Encoding.toXrefKey(targetRoomModelId, targetRoom.k);
+    const targetRoomKey = targetRoom[QC.Key];
+    const parentXref = Encoding.toXrefKey(targetRoomModelId, targetRoomKey);
     // creeate new stream
-    const streamId = await createStream(token,
-        defaultModel.modelId,
+    const streamId = await client.createStream(defaultModel.modelId,
         roomName,
         uniformatClassId,
         categoryId,
         classification,
         parentXref, // becuse stream is assigned to room we use same key for host & room
         parentXref, 
-        targetLevel.k);
+        targetLevel[QC.Key]);
```

Comment:
I think at this point we can add all these positional arguments to the props object, that would hold all of them.